pwa/src/components/applications/applicationForm.tsx

```diff
@@ -72,17 +73,17 @@ export default function ApplicationForm({ id }) {
 
     let body: {} = {
       name: event.target.name.value,
-      description: event.target.description ? event.target.description.value : null,
-      public: event.target.public.value ? event.target.public.value : null,
-      secret: event.target.secret.value ? event.target.secret.value : null,
-      resource: event.target.resource.value ? event.target.resource.value : null,
+      description: event.target.description
+        ? event.target.description.value : null,
+      public: event.target.public.value
+        ? event.target.public.value : null,
+      secret: event.target.secret.value
+        ? event.target.secret.value : null,
+      resource: event.target.resource.value
+        ? event.target.resource.value : null,
+      domains,
     };
 
-    if (domains.length !== 0) {
-      body["domains"] = domains;
-    } else {
-      body["domains"] = [];
-    }
```

Comment:
Is dit bewust weggehaald? Ik weet niet zo goed wat het deed.

Reply:
ja dit is goed en werk dat goed